cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionDiffCommand.java

```diff
@@ -147,6 +149,13 @@ public class ExtractionDiffCommand extends Command {
       description = "Optional username who owns the branch when pusing to a repository")
   String pushToBranchCreatedBy;
 
+  @Parameter(
+      names = {"--push-to-branch-notifiers", "-pbn"},
+      variableArity = true,
+      required = false,
+      description = "Optional list of notifiers when pusing to a repository")
```

Comment:
Could we add a bit more information for CLI users as to what these notifiers are for and when they would get called/invoked in the localization process?

Reply:
adding some basic info, but this would require a full independent documentation ideally